From a5d0d643e33652ef6bcae70e1884e773bbf213a0 Mon Sep 17 00:00:00 2001
From: Fernando Guzman Lugo <x0095840@ti.com>
Date: Sat, 27 Mar 2010 02:01:20 -0600
Subject: [PATCH 076/135] DSPBRIDGE: remove unused module parameters

This patch removes unused module parameters:
-phys_mempool_base
-phys_mempool_base

Signed-off-by: Fernando Guzman Lugo <x0095840@ti.com>
---
 drivers/dsp/bridge/rmgr/drv_interface.c |   24 +++---------------------
 1 files changed, 3 insertions(+), 21 deletions(-)

diff --git a/drivers/dsp/bridge/rmgr/drv_interface.c b/drivers/dsp/bridge/rmgr/drv_interface.c
index cbc1767..bd98605 100644
--- a/drivers/dsp/bridge/rmgr/drv_interface.c
+++ b/drivers/dsp/bridge/rmgr/drv_interface.c
@@ -88,8 +88,6 @@ static s32 driver_major;
 static char *base_img;
 char *iva_img;
 static s32 shm_size = 0x500000;	/* 5 MB */
-static u32 phys_mempool_base;
-static u32 phys_mempool_size;
 static int tc_wordswapon;	/* Default value is always false */
 #ifdef CONFIG_BRIDGE_RECOVERY
 #define REC_TIMEOUT 5000	/*recovery timeout in msecs */
@@ -133,13 +131,6 @@ MODULE_PARM_DESC(base_img, "DSP base image, default = NULL");
 module_param(shm_size, int, 0);
 MODULE_PARM_DESC(shm_size, "shm size, default = 4 MB, minimum = 64 KB");
 
-module_param(phys_mempool_base, uint, 0);
-MODULE_PARM_DESC(phys_mempool_base,
-		 "Physical memory pool base passed to driver");
-
-module_param(phys_mempool_size, uint, 0);
-MODULE_PARM_DESC(phys_mempool_size,
-		 "Physical memory pool size passed to driver");
 module_param(tc_wordswapon, int, 0);
 MODULE_PARM_DESC(tc_wordswapon, "TC Word Swap Option. default = 0");
 
@@ -321,18 +312,9 @@ static int __devinit omap34_xx_bridge_probe(struct platform_device *pdev)
 	}
 	dev_dbg(bridge, "%s: requested shm_size = 0x%x\n", __func__, shm_size);
 
-	if (pdata->phys_mempool_base && pdata->phys_mempool_size) {
-		phys_mempool_base = pdata->phys_mempool_base;
-		phys_mempool_size = pdata->phys_mempool_size;
-	}
-
-	dev_dbg(bridge, "%s: phys_mempool_base = 0x%x \n", __func__,
-		phys_mempool_base);
-
-	dev_dbg(bridge, "%s: phys_mempool_size = 0x%x\n", __func__,
-		phys_mempool_base);
-	if ((phys_mempool_base > 0x0) && (phys_mempool_size > 0x0))
-		mem_ext_phys_pool_init(phys_mempool_base, phys_mempool_size);
+	if ((pdata->phys_mempool_base > 0) && (pdata->phys_mempool_size > 0))
+		mem_ext_phys_pool_init(pdata->phys_mempool_base,
+						pdata->phys_mempool_size);
 	if (tc_wordswapon) {
 		dev_dbg(bridge, "%s: TC Word Swap is enabled\n", __func__);
 		reg_set_value(TCWORDSWAP, (u8 *) &tc_wordswapon,
-- 
1.7.0.4

